### PR TITLE
Version updates for 1.1.6

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Template updates for 1.1.6.

### CODE REVIEW

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj [hunk 1]
```diff
@@ -15,10 +15,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" Condition="'$(IncludeApplicationInsights)' == 'True'" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.0.7" Condition="'$(Framework)' != 'netcoreapp1.1'" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.4" Condition="'$(Framework)' == 'netcoreapp1.1'" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.5" Condition="'$(Framework)' == 'netcoreapp1.1'" />
```
Comment (JunTaoLuo) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj]: This file looks odd compared to the other files. Every other project has two separate dependency sections for 1.0 and 1.1. However, the only PackageReference that's different between 1.0 and 1.1 for this project is `Microsoft.AspnetCore`. Does this mean that the 1.0 fsproj also references Microsoft.AspNetCore.Mvc 1.1.6 and Microsoft.Extensions.Logging.Debug 1.1.2?

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj [hunk 2]
```diff
@@ -15,10 +15,10 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.0.0" Condition="'$(IncludeApplicationInsights)' == 'True'" />
     <PackageReference Include="Microsoft.AspNetCore" Version="1.0.7" Condition="'$(Framework)' != 'netcoreapp1.1'" />
-    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.4" Condition="'$(Framework)' == 'netcoreapp1.1'" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.4" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.5" Condition="'$(Framework)' == 'netcoreapp1.1'" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.6" />
```
Comment (JunTaoLuo) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj]: This file looks odd compared to the other files. Every other project has two separate dependency sections for 1.0 and 1.1. However, the only PackageReference that's different between 1.0 and 1.1 for this project is Microsoft.AspnetCore. Does this mean that the 1.0 fsproj also references Microsoft.AspNetCore.Mvc 1.1.6 and Microsoft.Extensions.Logging.Debug 1.1.2?

Comment (JunTaoLuo) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj]: I verified that this looks really wrong by using the latest 1.1.5 sdk from dot.net. I ran the command `dotnet.exe new webapi -lang F# --framework netcoreapp1.0` and it gave me this fsproj:
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp1.0</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Controllers/*.fs" />
    <Compile Include="Startup.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore" Version="1.0.7" />
    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.4" />
    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.2" />

  </ItemGroup>
  
</Project>
```

This is not a supported configuration (mix of 1.0 and 1.1 packages).

Comment (muratg) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/WebApi-FSharp/Company.WebApplication1.fsproj]: @DamianEdwards FYI

**Comment by JunTaoLuo** [comment]
My previous comments keep getting collapsed but I think I found an issue where the F# 1.0 webapi projects were referencing 1.1 packages, which I'm trying to fix in this PR.

**Comment by mlorbetske** [comment]
lgtm

**Pull request merged**